lib/src/data/simulations.dart

```diff
@@ -86,6 +88,15 @@ class Simulations with ChangeNotifier {
         infoLink: 'https://en.wikipedia.org/wiki/Fourier_series',
         fav: _favorites[5],
       ),
+      SimulationCard(
+        id: 6,
+        simulationName: 'Pi Approximation',
```

Comment:
Simulation name is very generic, would be better if you could change it to something specific